<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_2</class>
 <widget class="QMainWindow" name="MainWindow_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1908</width>
    <height>886</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border:none;
background-color:transparent;
padding:0;
margin:0;
color:#fff;
}

#centralwidget{
background-color:#1f232a;
}










#leftmenusubcontainer{
background-color:#16191d;
}

#leftmenusubcontainer QPushButton{

padding:5px 10px;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
}

#headercontainer{
background-color:#2c313c;
}

#frame_6{
padding:5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="mainbodycontainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="mainbodycontainer_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>0</y>
            <width>1931</width>
            <height>81</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{
background-color:#131519;
}</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>750</x>
             <y>10</y>
             <width>431</width>
             <height>51</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string> SWARMULATOR</string>
           </property>
          </widget>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>91</width>
             <height>1291</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
background-color:#131519;
}</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>71</width>
            <height>1081</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{
background-color:#131519;
}</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>1850</x>
            <y>0</y>
            <width>71</width>
            <height>1101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{
background-color:#131519;
}</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>140</y>
            <width>651</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Comparative Analysis Of Swarm Algorithms:</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>250</y>
            <width>541</width>
            <height>301</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_5{
 border: 2px solid #2707de;
border-radius: 4px;
}</string>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>10</y>
             <width>211</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Accuracy</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Whale optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Coati optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>GreyWolf  optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>240</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Artificial Bee Colony</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>60</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Train</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>60</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_6" native="true">
          <property name="geometry">
           <rect>
            <x>710</x>
            <y>250</y>
            <width>541</width>
            <height>301</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_6
{
 border: 2px solid #2707de;
border-radius: 4px;
}</string>
          </property>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>10</y>
             <width>211</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Precision</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Whale optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Coati optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>GreyWolf  optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_22">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>240</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Artificial Bee Colony</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_23">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>60</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Train</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>60</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_7" native="true">
          <property name="geometry">
           <rect>
            <x>1280</x>
            <y>250</y>
            <width>541</width>
            <height>301</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_7
{
 border: 2px solid #2707de;
border-radius: 4px;
}</string>
          </property>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>10</y>
             <width>211</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Recall</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_25">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Whale optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_26">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Coati optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_27">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>GreyWolf  optimization</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_28">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>240</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Artificial Bee Colony</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_29">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>60</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Train</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_30">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>60</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_8" native="true">
          <property name="geometry">
           <rect>
            <x>640</x>
            <y>630</y>
            <width>741</width>
            <height>311</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_8

{
 border: 2px solid #2707de;
border-radius: 4px;
}</string>
          </property>
          <widget class="QLabel" name="label_31">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>30</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Algorithms</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_32">
           <property name="geometry">
            <rect>
             <x>450</x>
             <y>20</y>
             <width>271</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Selected Features</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_33">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>681</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Whale Optimization Algorithm (WOA)</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_34">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>160</y>
             <width>681</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Coati Optimization Algorithm (COA)</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_35">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>210</y>
             <width>681</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Grey Wolf Optimization (GWO)</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_36">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>250</y>
             <width>681</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Artificial Bee Colony (ABC)</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="previous">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>800</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  border: 1.5px solid #2707de;
  border-radius: 5px;  /* Optional: Add rounded corners */
  padding: 1px 5
px;  /* Optional: Adjust padding for button text */
}</string>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1908</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
